src/components/iceCreamContainer.js

```diff
@@ -0,0 +1,28 @@
+import React, { useState } from 'react'
+import { connect } from 'react-redux'
+import { buyIceCream } from "../redux"
+
+function IceCreamContainer(props) {
+    const [number, setNumber] = useState()
```

Comment:
Again same mistake, you need to initialize the value to avoid undefined and null values.